Popup/Popup.module.less

```diff
@@ -84,6 +84,12 @@
 				padding: @agate-popup-closebutton-title-padding;

```

Comment:
I think we don't need this line.
this line make another bug in Gallium skin. 
the up and down padding is changed when Close button show.
Could you check this also? I saw this line was implemented by you.
There may be something I am missing.

Reply:
yes we don;t need this line. Also, in order to avoid the top and bottom paadding change in Gallium, a 0 padding had to be set for the title